src/db/models/JobModel.ts

```diff
@@ -177,4 +191,20 @@ export class JobModel extends Model<JobModel> {
     onDelete: "CASCADE",
   })
   applications: ApplicationModel[];
+
+  @AfterCreate
```

Comment:
1. use `AfterUpdate`. even in that send mail only if previous is `Active=false` and now its `Active=true`. 
2. Recipients should be all eligible students. so first get those and then send them.
3. feel free to discuss eligibility of students with @cse210001015

Reply:
Actually this mail is meant for admin when there is an entry in Job Table
Students receive mail when there is an entry in offer Table